<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Data</class>
 <widget class="QDialog" name="Data">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Data</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QObject{background-color: #3b5998;}</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>90</y>
     <width>691</width>
     <height>511</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame{
background-color: rgb(83, 86, 173);
border: 5px solid white;
margin: 20% 0;}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>-20</y>
      <width>301</width>
      <height>531</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox{background-color: rgb(255, 255, 255);}
QLabel{
	font: 12pt &quot;Segoe UI&quot;;
}</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLineEdit" name="lineEdit_14">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>40</y>
       <width>221</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);
border:none;
background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>MY DATA</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_fname">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>90</y>
       <width>201</width>
       <height>91</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{
border: 2px solid #6b6b6b ;
background-color: rgb(255, 255, 255);
}</string>
     </property>
     <property name="text">
      <string>First Name</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_lname">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>160</y>
       <width>201</width>
       <height>91</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{
border: 2px solid #6b6b6b ;
background-color: rgb(255, 255, 255);
}</string>
     </property>
     <property name="text">
      <string>Last Name</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_datebirth">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>230</y>
       <width>201</width>
       <height>91</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{
border: 2px solid #6b6b6b ;
background-color: rgb(255, 255, 255);
}</string>
     </property>
     <property name="text">
      <string>Birth Date</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_email">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>300</y>
       <width>201</width>
       <height>91</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{
border: 2px solid #6b6b6b ;
background-color: rgb(255, 255, 255);
}</string>
     </property>
     <property name="text">
      <string>Email</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_phone">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>370</y>
       <width>201</width>
       <height>91</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{
border: 2px solid #6b6b6b ;
background-color: rgb(255, 255, 255);
}</string>
     </property>
     <property name="text">
      <string>Phone</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>467</y>
       <width>120</width>
       <height>42</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI Black</family>
       <pointsize>12</pointsize>
       <italic>false</italic>
       <fontweight>Black</fontweight>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
    width: 100px;
    background: #ee0979;
    font-weight: bold;
    color: white;
    border: 0 none;
    border-radius:20px;
    padding: 7px 5px;
	font: 900 12pt &quot;Segoe UI Black&quot;;
}


QPushButton::hover{
   border: 2px solid white;
    width: 100px;
    background: #ee0979;
    font-weight: bold;
    color: white;
     border-radius:20px;
    padding: 7px 5px;
	font: 900 12pt &quot;Segoe UI Black&quot;;
}
</string>
     </property>
     <property name="text">
      <string>Show</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
